pkg/ipam/clusterpool.go

```diff
@@ -490,7 +490,6 @@ func newCRDWatcher(conf Configuration, nodeWatcher nodeWatcher, owner Owner, nod
 
 	// Subscribe to CiliumNode updates
 	nodeWatcher.RegisterCiliumNodeSubscriber(c)
-	owner.UpdateCiliumNodeResource()
```

Comment:
This seems unnecessary as we already do this "block on CiliumNode creation" in daemon.go (line 888).